scripts/fdbshow

```diff
@@ -87,6 +87,7 @@ class FdbShow(object):
         if not fdb_str:
             return
 
+        bvid_tlb = {}
```

Comment:
if you meant bvid table, please rename to `bvid_tbl`

Reply:
I meant Translation lookaside buffer, as in MMU, thus TLB.